spring-kafka-test/src/main/java/org/springframework/kafka/test/context/EmbeddedKafka.java

```diff
@@ -53,8 +53,9 @@
  * @author Elliot Metsger
  * @author Zach Olauson
  * @author Gary Russell
+ * @author Sergio Lourenco
  *
- * @since 1.3
+ * @since 2.24
```

Comment:
No, you can't change a `@since`. You just have to add it into the method Javadoc as I mentioned on the line.